### ISSUE TITLE
lint for detecting objects with identical attribute names

### ISSUE THREAD

**Issue by ext11n7** [objectionary/lints#997]
## Problem

Currently, there is no lint that detects when two or more attributes of the same EO object share identical names. Duplicate attribute names lead to ambiguity and compilation errors that are hard to trace.

## Suggestion

Add a new lint `duplicate-attribute-names` that reports a `ERROR` level defect when an object declares two or more attributes with the same name.

## Example of code that should trigger the lint

```
[x x y] +> my-object
  ...
```

Here `x` is declared twice, which is a mistake.

## Expected behavior

The lint should report an `ERROR` level defect pointing to the duplicated attribute name and the line where it appears.

**Comment by ext11n7** [comment]
Closing this — found that duplicate-names lint already exists in the project

**Comment by 0crat** [comment]
@ext11n7 Thanks for submitting this ticket! However, as you see, it was not accepted by the team, which is sad. You've earned -8 points. By reporting bugs or suggesting enhancements, you help our project improve its quality. Please, don't be discouraged. Instead, try to be more careful next time, and [do not hesitate](https://www.yegor256.com/2014/04/13/bugs-are-welcome.html) to [report](https://www.yegor256.com/2018/04/24/right-way-to-report-bugs.html) bugs. Your running score is [+12](https://www.eolang.org/zerocracy/objectionary-vitals.html); don't forget to check your [Zerocracy account](https://www.zerocracy.com/) too).
